.flake8

```diff
@@ -0,0 +1,40 @@
+[flake8]
+exclude =
+    .git,
+    .gitignore,
+    *.pot,
+    *.py[co],
+    __pycache__,
+    venv,
+    .env
+
+ignore =
```

Comment:
It'd be nice to add information about why given rules are excluded from flake and what rules are represented by this codes.